models/stations.go

```diff
@@ -69,6 +69,7 @@ type Station struct {
 	LastUpdate      time.Time          `json:"last_update" bson:"last_update"`
 	Functions       []Function         `json:"functions" bson:"functions"`
 	IsDeleted       bool               `json:"is_deleted" bson:"is_deleted"`
+	Tags            []Tag              `json:"tags"`
```

Comment:
this is the struct saved in mongo it is not good to add tags in here because it will be saved in mongo